jmetal-core/src/main/java/org/uma/jmetal/problem/impl/AbstractFlexibleVariableProblem.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2018 Jackson Antonio do Prado Lima <jacksonpradolima at gmail.com>.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.uma.jmetal.problem.impl;
+
+import org.uma.jmetal.solution.IFlexibleVariableSolution;
+
+/**
+ *
+ * @author Jackson Antonio do Prado Lima <jacksonpradolima at gmail.com>
+ * @version 1.0
+ * @param <S> Solution type
+ */
+@SuppressWarnings("serial")
+public abstract class AbstractFlexibleVariableProblem<S extends IFlexibleVariableSolution<?>> extends AbstractGenericProblem<S> {
```

Comment:
This implementation does not do what we expect from it. `AbstractGenericProblem` does provide setters, but they are not bounded by the min/max we define here. The limits are just totally ineffective.

Like you had to redo the implementation of `Solution`, you need to redo the one for `Problem`. What you can do, however, is to make the existing implementations extend your own, which would drastically reduce code redundancy.